daemon/call.c

```diff
@@ -208,6 +205,12 @@ static void call_timer_iterator(void *key, void *val, void *ptr) {
 		goto delete;
 	}
 
+	// other timeouts not applicable to foreign calls
+	if (IS_FOREIGN_CALL(c)) {
+		c->deleted = c->ml_deleted = poller_now + cm->conf.delete_delay;
```

Comment:
As @pkuzak said, that part of the code is a remnant from an old idea. From what I remember, the idea was that if a rtpengine machine became responsible for a foreign call (i.e. started receiving media for that call) to also consider the DELETE_DELAY from the point which no media was being received. But we decided to keep it simple and terminate foreign calls only via normal DEL notifications, FINAL_TIMEOUT and cli commands.